spec/support/features/document_capture_step_helper.rb

```diff
@@ -3,8 +3,9 @@ def submit_images
     click_on 'Submit'
 
     # Wait for the the loading interstitial to disappear before continuing
-    expect(page).to have_content(t('doc_auth.headings.interstitial'))
-    expect(page).not_to have_content(t('doc_auth.headings.interstitial'), wait: 10)
+    if page.has_content?(t('doc_auth.headings.interstitial'))
+      expect(page).not_to have_content(t('doc_auth.headings.interstitial'), wait: 10)
+    end
```

Comment:
Is the `if` necessary here? I'd imagine it could work in either case, i.e. immediately pass if it was already not present, or wait up to 10 seconds if it is.

```suggestion
    expect(page).not_to have_content(t('doc_auth.headings.interstitial'), wait: 10)
```

Reply:
good call! fixed it in c68b705a339ef9974b415d2f6b2fd93e4e9533be